lib/WebSocket.js

```diff
@@ -98,10 +98,15 @@ class WebSocket extends EventEmitter {
   }
 
   set binaryType (type) {
-    if (type === 'arraybuffer' || type === 'nodebuffer') {
-      this._binaryType = type;
-    } else {
-      throw new SyntaxError('unsupported binaryType: must be either "nodebuffer" or "arraybuffer"');
+    if (constants.BINARY_TYPES.indexOf(type) < 0) {
+      throw new SyntaxError('unsupported binaryType: must be either "' +
```

Comment:
This is a breaking change as the error message is changed but I **think** there is not much code that relies on this. We can try to search for this error message on Github.

I think it's better to remove the error completely. The browser no longer throw this error if an invalid `binaryType` is specified. We should do the same and fail silently if an invalid `binaryType` is supplied.